src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -15,17 +15,14 @@
             <GroupBox x:Uid="AnalysisGroupBox"
                       Header="{x:Static local:AdvancedOptionPageStrings.Option_Analysis}">
                 <StackPanel>
-                    <Label Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_analysis_scope}"/>
-                    <StackPanel Margin="15, 0, 0, 0">
-                        <RadioButton  GroupName="Background_analysis_scope"
-                                              x:Name="Background_analysis_scope_active_file"
-                                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Active_File}"/>
-                        <RadioButton  GroupName="Background_analysis_scope"
-                                              x:Name="Background_analysis_scope_open_files"
-                                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Open_Files_And_Projects}"/>
-                        <RadioButton  GroupName="Background_analysis_scope"
-                                              x:Name="Background_analysis_scope_full_solution"
-                                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution}"/>
+                    <Label Content="{x:Static local:AdvancedOptionPageStrings.Option_Run_background_code_analysis_for}"/>
+                    <StackPanel>
+                        <ComboBox x:Name="Run_background_code_analysis_for" IsEditable="false" AutomationProperties.Name="{x:Static local:AdvancedOptionPageStrings.Option_Run_background_code_analysis_for}">
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_None}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_None_Tag}" />
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Active_File}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Active_File_Tag}" />
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Open_Files}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Open_Files_Tag}" />
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution_Tag}" />
```

Comment:
So i'm ok with this UI.  HOwever, i think we should have a textbox next to it (or below) that explains waht each means.  .ActiveFile *esp* is really going to be confusing as we will absolutely be doing BG analysis and reporting errors for the non-active-file.

Reply:
@mikadumont ?

Reply:
In the UX meeting, David suggested to explicitly change the textbox title to `Run background code analysis for:` which should make it clear to the users that the option applies only to analyzers, not compiler errors. https://github.com/dotnet/roslyn/pull/57172#issuecomment-970105416 has the screenshot.